src/bridge.ts

```diff
@@ -327,3 +327,80 @@ export const createOpenAPIGraphQLBridge = (config: CreateOpenAPIGraphQLBridgeCon
       createOpenAPISchemaWithValidate(operations, config),
   }
 }
+
+export const transformer = () => {
```

Comment:
The function is currently not in use. I would move it into a test, as this is an actual implementation of a request transformation.

Before we can test the functionality though, we have to implement support for this signature.

The request transformation should happen when the `expressMiddleware` is created (see `CreateMiddlewareConfig`). So you have to extend the function `createExpressMiddleware` (see https://github.com/freshcells/graphql-rest-converter/blob/5734b021d61166b4487fc750004e952c3b6eaf6c/src/bridge.ts#L194-L194).